stream_interfaces.go

```diff
@@ -49,7 +76,22 @@ type ClientStreamingClient[Req any, Res any] interface {
 // message stream and the type of the unary response message. It is used in
 // generated code.
 type ClientStreamingServer[Req any, Res any] interface {
+	// Recv reads a request message from the client. This method is called repeatedly
+	// to receive all messages sent by the client as part of the request stream.
+	// by the Req type parameter of the ClientStreamingServer. If an error occurs on
```

Comment:
nit: no need to mention `by the Req type parameter of the ClientStreamingServer.`